app/_src/gateway/kong-manager/strengthen-security.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Strengthen Security in Kong Manager
+badge: enterprise
+---
+
+{% if_version gte:3.10.x %}
```

Comment:
```suggestion
```
You don't need this for the page - adding the page to the nav file for 3.10 will only generate it for that version.